debian/leap-mx.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
Maybe we should add this file to `pkg/leap-mx.service` as we have already the `pkg/leap-mx.init`. This will mean split this in two commits one for the `develop` branch with the service file and another for the debian branch with the control.